client/containers/Pub/SpubHeader/SpubHeader.tsx

```diff
@@ -38,90 +28,76 @@ const SpubHeader = () => {
 				abstract: newAbstract,
 			},
 		}));
-		return apiFetch.put('/api/submissions', {
-			abstract: newAbstract,
-			id: submission.id,
-		});
+		return pendingPromise(
+			apiFetch.put('/api/submissions', {
+				abstract: newAbstract,
+				id: submission.id,
+			}),
+		);
 	};
 
 	const updateAndSavePubData = async (newPubData: Partial<PubPageData>) => {
 		updatePubData(newPubData);
-		return apiFetch('/api/pubs', {
-			method: 'PUT',
-			body: JSON.stringify({
-				...newPubData,
-				pubId: pubData.id,
-				communityId: pubData.communityId,
-			}),
-		}).catch(() => updatePubData(pubData));
+		return pendingPromise(
+			apiFetch('/api/pubs', {
+				method: 'PUT',
+				body: JSON.stringify({
+					...newPubData,
+					pubId: pubData.id,
+					communityId: pubData.communityId,
+				}),
+			}).catch(() => updatePubData(pubData)),
+		);
 	};
 
 	const setSelectedTab = (nextTab: SpubHeaderTab) => {
 		updateSubmissionState({ selectedTab: nextTab });
 	};
 
-	const updateHistoryData = (newHistoryData: Partial<PubHistoryState>) => {
-		return updateLocalData('history', newHistoryData);
-	};
-
-	const instructionTabTitle = renderTabTitle('align-left', 'Instructions');
-	const submissionTabTitle = renderTabTitle('manually-entered-data', 'Submission');
-	const previewTabTitle = renderTabTitle('eye-open', 'Preview & Submit');
-	const contributorsTabTitle = renderTabTitle('people', 'Contributors');
-	const maybeActiveClass = (tabId: string) => (tabId === selectedTab ? 'active' : 'inactive');
-
 	return (
-		<Tabs
-			id="spubHeader"
-			onChange={(t: SpubHeaderTab) => setSelectedTab(t)}
-			selectedTabId={selectedTab}
-			className="spub-header-component tabs bp3-large"
-		>
-			<Tab
-				id="instructions"
-				title={instructionTabTitle}
-				className={`tab-panel ${maybeActiveClass('instructions')}`}
-				panel={
-					<InstructionsTab
-						submissionWorkflow={submission.submissionWorkflow}
-						onBeginSubmission={() => setSelectedTab('submission')}
-					/>
-				}
-			/>
-			<Tab
-				id="submission"
-				title={submissionTabTitle}
-				className={`tab-panel submission ${maybeActiveClass('submission')}`}
-				panel={
-					<SubmissionTab
-						abstract={submission.abstract}
-						onUpdatePub={updateAndSavePubData}
-						onUpdateAbstract={updateAbstract}
-						pub={pubData}
-					/>
+		<div className="spub-header-component">
+			<SpubHeaderToolBar
+				onSelectTab={(t: SpubHeaderTab) => setSelectedTab(t)}
+				selectedTab={selectedTab}
+				showSubmitButton={
+					submission.status === 'incomplete' && selectedTab !== 'instructions'
 				}
+				submission={submission}
+				status={submission.status as SubmissionStatus}
```

Comment:
This is a quirk of `SpubHeaderToolbar` you inherited from #1856, but I notice that this prop is redundant since we pass the entire `submission` in as well.